docs/conf.py

```diff
@@ -13,12 +13,12 @@
 import os
 import sys
 
-sys.path.insert(0, os.path.abspath('../../packages'))
+sys.path.insert(0, os.path.abspath('../packages'))
```

Comment:
Use `__file__` to clean the relative import

Reply:
Done!